yamscripts.yml

```diff
@@ -41,7 +41,7 @@ scripts:
     # test:jpm: Run jpm tests
     jpm: jpm test -b beta
     # test:karma: Run content tests only
-    karma: karma start
+    karma: karma start --browsers FirefoxNightly
```

Comment:
So this is just overriding the `browsers` set in the [karma.conf.js:7](https://github.com/mozilla/activity-streams/blob/fix-empty-case/karma.conf.js#L7)?